atlasdb-commons/src/test/java/com/palantir/util/MutuallyExclusiveSetLockTest.java

```diff
@@ -79,8 +79,7 @@ public void testSimpleNotBlock() throws Exception {
             thread = createThread(mutuallyExclusiveSetLock, Arrays.asList("heyo"));
             thread.setDaemon(true);
             thread.start();
-            Thread.sleep(100);
-            Assert.assertFalse(thread.isAlive());
+            thread.join(60 * 1000);
```

Comment:
Why not use the same timeout?  60s is a while to wait for a test to fail.  Was it flickering with the 100ms timeout?